build/build.xml

```diff
@@ -101,6 +108,17 @@ External Dependencies:
 		<property name="src.isInit" value="true" />
 	</target>
 
+	<!-- create script to run built commandbox using sources (for core cfml development) -->

```

Comment:
For what it's worth, when I want to just hack on the CFML bits, I have a batch file I use to create a symlink from the CFML folder to my Git repo.